### PR TITLE
Feature/dfs 38 latest activity

### PR DESCRIPTION
**Pull request by AnHere**

### CODE REVIEW

File: lib/hooks/use_user_asset_data.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:neo/services/data_service.dart';
+import 'package:neo/services/stockinvestment_service.dart';
+import 'package:neo/types/data_container.dart';
+import 'package:neo/types/investment/investment_data.dart';
+
+import '../services/stockdata_service.dart';
+
+DataContainer<InvestmentData> useInvestmentData(String symbol) {
```
Comment (TilDoesCode) [inline lib/hooks/use_user_asset_data.dart:9]: rename to useUserAssetsForSymbol to better line up with the naming of other hooks

File: lib/services/stockinvestment_service.dart [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      //Der Nutzer kauft einen Tag später noch eine Apple Aktie
+      UserassetDatapoint(
+          tokenAmmount: 3.0,
+          symbol: "dAAPL",
+          currentPrice: 100,
+          time: DateTime(2022, 6, 16, 12, 0),
+          difference: 1.0),
+    ];*/
+
+class StockInvestmentUtil {
```
Comment (TilDoesCode) [inline lib/services/stockinvestment_service.dart:31]: Could be redundant with logic I implemented for the portfolio screen. We will have to check this later.

File: lib/widgets/filter/valuefilter_widget.dart [hunk 3]
```diff
@@ -5,8 +5,8 @@ import 'package:neo/style/theme.dart';
 
 class ValueFilter extends HookWidget {
   final String text;
-  final String value;
-  final String currentValue;
+  final Object value;
+  final Object currentValue;
```
Comment (TilDoesCode) [inline lib/widgets/filter/valuefilter_widget.dart:9]: Type this

**Review by TilDoesCode — CHANGES_REQUESTED** [review]

File: lib/widgets/filter/valuefilter_widget.dart (hunk 3 above)
Comment (AnHere) [reply lib/widgets/filter/valuefilter_widget.dart:9]: I dont get it. "final this value"?

File: lib/widgets/cards/open_order_card.dart [hunk 4]
```diff
@@ -4,8 +4,10 @@ import 'package:flutter/material.dart';
 import 'package:neo/widgets/development_indicator/small_change_indicator.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
+// ignore: must_be_immutable
```
[the remark this answers is not in the document]
Comment (AnHere) [reply lib/widgets/cards/open_order_card.dart:7]: What do you mean? Do I need to remove this comment? I guess it is not my implementation, I only moved the file :D

**Review by TilDoesCode — APPROVED** [review]

**Pull request merged**